index.html

```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <script src="js/index.js"></script>
+
+    <div id="titlewrap">
+      <h1 id="title">freeCodeCamp Survey Form</h1>
+      <p id="description">
+        Thank you for taking the time to help us improve the platform
+      </p>
+    </div>
+    <div id="myForm">
+      <form id="survey-form">
+        <div class="div1">
+          <label id="name-label">Name </label><br />
+          <input type="text" id="name" placeholder="Enter your name" />
+        </div>
+        <div class="div1">
+          <label id="email-lable">Email</label><br />
+          <input
+            type="email"
+            id="email"
+            pattern=".+@globex\.com"
+            required
+            placeholder="Enter your email"
+          />
+        </div>
+     <div class="div1">
+      <label id="number-label">Age (Optional)</label><br />
+      <input
+        type="number"
+        id="number"
+        pattern="[0-9]"
+        min="0"
+        max="100"
+        required
+        placeholder="Enter your age"
+      />
+     </div>
+       
+       <div class="div1">
+        <label>Current Role</label><br />
+        <select id="dropdown">
+          <option value="student">Student</option>
+          <option value="teacher">Teacher</option></select>
+       </div>
+       
+       <div class="div1">
+        <label>Would you recommend freeCodeCamp to a friend?</label><br />
+        <label>
+          <input type="radio" name="user-recommendation" value="Maybe" />
+          May be
+        </label>
+        <br />
+        <label>
+          <input type="radio" name="user-recommendation" value="Sure" />
+          Sure</label>
+          </div>
+          <div class="div1">
+        <label
+          >What would you like to see improved? (Check all that apply)</label
+        ><br />
+        <label>
+          <input type="checkbox" name="prefer" value="Front-end Project" />
+          Front-end Project </label
+        ><br />
+        <label>
+          <input type="checkbox" name="prefer" value="Back-end Project" />
+          Back-end Project
+        </label>
+        </div>
+        <div class="div1">
+        <label>Any comments or suggestions?</label>
+        </div>
+        <div class="div1">
+        <input type="textarea" style="height: 70px;
+        width: -webkit-fill-available;" placeholder="Enter your comment" >
+      </div>
+      <div class="div1">
+        <button type="submit" id="submit">Submit</button>
+      </div>
```

Comment:
- [ ] I kindly suggest you consider CSS styling such as _flexbox_  to great the necessary alignment on your project instead of redundant `br` tags throughout your code.